sdks/python/apache_beam/transforms/enrichment_handlers/bigtable.py

```diff
@@ -63,32 +65,43 @@ class EnrichWithBigTable(EnrichmentSourceHandler[beam.Row, beam.Row]):
       to use as `row_key` for BigTable querying.
     row_filter: a ``:class:`google.cloud.bigtable.row_filters.RowFilter``` to
       filter data read with ``read_row()``.
+      Defaults to `CellsColumnLimitFilter(1)`.
+    app_profile_id (str): App profile ID to use for BigTable.
```

Comment:
Would be good to link to https://cloud.google.com/bigtable/docs/app-profiles here

Reply:
added!